ister.py

```diff
@@ -408,43 +411,62 @@ def get_current_format():
         frmt = format_file.read().strip()
     return frmt
 
+def add_default_dnf_config(args):
```

Comment:
I'm inclined to not have this built into ister and just let dnf use /etc/dnf.conf if it exists by default or else error early in the process around validate template.

Reply:
Or if the dnf-conf passed in doesn't exist still error.

Reply:
Having this built-in would allow for a similar default for dnf analogous to what swupd has with its content and version URLs.  It's kinda ugly though, but maybe less ugly than the alternative of passing repo URLs and then generating a conf in /tmp somewhere?

Reply:
So the built-ins for swupd urls are essentially legacy.
It would be nice to remove them and None the argument by default. Then we can only add those arguments to swupd when they are provided instead of every time.

Reply:
Ok.  I'll do a separate commit for this as pre-work for this commit.